data/tabular/SIDER/meta.yaml

```diff
@@ -0,0 +1,206 @@
+---
+name: SIDER
+description: Database of marketed drugs and adverse drug reactions (ADR), grouped into 25 system organ classes.
+identifiers:
+    - id: SMILES
+      type: SMILES
+      description: SMILES
+targets:
+    - id: hepatobiliary_disorders
+      description: hepatobiliary disorders
+      type: boolean
+      names:
+          - noun: hepatobiliary disorders
+    - id: metabolism_and_nutrition_disorders
+      description: metabolism and nutrition disorders
+      type: boolean
+      names:
+          - noun: metabolism and nutrition disorders
+    - id: eye_disorders
+      description: product issues
+      type: boolean
+      names:
+          - noun: product issues
+    - id: investigations
+      description: eye disorders
+      type: boolean
+      names:
+          - noun: eye disorders
+    - id: musculoskeletal_and_connective_tissue_disorders
+      description: investigations
+      type: boolean
+      names:
+          - noun: investigations
```

Comment:
isn't there a mismatch between your column names and the `names`? 
They seem one off. 

Generally, it is probably better if we can manage to get multiple names to increase the diversity of the training data

Reply:
fixed this now, added one noun to each